### PR TITLE
Update CODEOWNERS and REVIEWERS to use the Wonder Blocks team.

### PR DESCRIPTION
**Pull request by jeresig**
## Summary:
Previously we were tagging individuals, which doesn't feel right. Instead let's use the Wonder Blocks team for this!

Issue: XXX-XXXX

## Test plan:

### CODE REVIEW

File: .github/REVIEWERS [hunk 1]
```diff
@@ -50,4 +50,4 @@ Regex Examples:
 [ON PULL REQUEST] (DO NOT DELETE THIS LINE)
 
 # Make these people reviewers for any changes made to any file
-**/*                @Khan/frontend-infra!
+**/*                @Khan/wonder-blocks!
```
Comment (jandrade) [inline .github/REVIEWERS:53]: Good call!

File: CODEOWNERS [hunk 2]
```diff
@@ -1,14 +1 @@
...
-packages/wonder-blocks-data/*   @somewhatabstract @jeresig
-packages/wonder-blocks-icon-button/*   @kevinbarabash @jeresig
-packages/wonder-blocks-link/*   @kevinbarabash @jeresig
-packages/wonder-blocks-modal/*   @jandrade @kevinbarabash @jeresig
-packages/wonder-blocks-popover/*   @jandrade @jeresig
-packages/wonder-blocks-testing/*   @somewhatabstract @jandrade @jeresig
-packages/wonder-blocks-timing/*   @somewhatabstract @jeresig
-packages/wonder-blocks-tooltip/*   @somewhatabstract @kevinbarabash @jeresig
-packages/wonder-blocks-typography/*   @briangenisio @kevinbarabash @jeresig
+*   @Khan/wonder-blocks
```
Comment (jandrade) [inline CODEOWNERS:1]: Out of curiosity... do you think we still need this file now that we use Gerald to notify the team?

**Review by jandrade — APPROVED** [review]
Thanks for simplifying this! Just added a question about the `CODEOWNERS` file.

File: CODEOWNERS (hunk 2 above)
Comment (jeresig) [reply CODEOWNERS:1]: I don't know - I guess we may not need it! We could just rely on Gerald... maybe I'll just remove this file instead!

**Review by somewhatabstract — APPROVED** [review]

**Pull request merged**